remoting/subsystem/src/main/java/org/jboss/as/remoting/HttpConnectorAdd.java

```diff
@@ -51,6 +52,14 @@ private HttpConnectorAdd() {
     }
 
     @Override
+    protected void populateModel(OperationContext context, ModelNode operation, Resource resource) throws OperationFailedException {
+        super.populateModel(context, operation, resource);
+        final PathAddress address = PathAddress.pathAddress(operation.get(OP_ADDR));
+        final String connectorName = address.getLastElement().getValue();
```

Comment:
shortcut for this is, context.getCurrentAddressValue();

Reply:
@ctomc thank you, I agree, I'll make this change in the next commit.